scripts/checkContingencies.py

```diff
@@ -8,8 +8,7 @@
 import xml.etree.ElementTree as ET
 
 ### Regexes ####################################################################
-contingency_event_time = 10
-timeline_log_regex = re.compile("^" + str(contingency_event_time) + "\s\|\s([\w-]+)\s\|\s\w+\s[:\s]*[\w\s]+$", re.MULTILINE)
+timeline_log_regex = re.compile("^10\.000\s\|\s([\w-]+)\s\|\s\w+\s[:\s]*[\w\s]+$", re.MULTILINE)
```

Comment:
I would like to keep the contingency_event_time as it helps understanding  the regex, and also be resilient to the precision if not equals to 1e-3